CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - Remove deprecated `configtest.LoadConfig` and `configtest.LoadConfigAndValidate` (#4659)
 - Move service.ConfigMapConverterFunc to config.MapConverterFunc (#4673)
   - Add context to config.MapConverterFunc (#4678)
+- Builder: the skip compilation should only be supplied as a CLI flag. Previously, it was possible to specify that in the YAML file, contrary to the original intention (#4645)
```

Comment:
This would be nice to include in the PR description/commit message, otherwise it is not clear why the change is made.